docs/BindingsGuide_Java.md

```diff
@@ -121,7 +153,15 @@ _TODO_
 
 ### SQLOutput Attribute
 
-_TODO_
+In the Java functions runtime library, use the @SQLOutput annotation (com.microsoft.azure.functions.sql.annotation.SQLOutput) on parameters whose values you want to upsert into the target table. This annotation supports the following elements:
+
+| Element |Description|
+|---------|---------|
+|**name** |  Required. The name of the variable that represents the query results in function code. | 
```

Comment:
This isn't correct (there are no "query results" in output bindings)

@lucyzhang929 We should probably just use the same descriptions you added in your PR

https://github.com/Azure/azure-functions-sql-extension/pull/481/files#diff-07386b4163c1406052b8b902fb2322202c7a235a720c958e540ea4f76fe53f0aR21

That sound good to you?

Reply:
Followed up and made it `Required. The variable name used in function.json.` as reflected in the PR Lucy added.